python/ray/rllib/agents/sac/sac/rllib_proxy/_patched/_trainer.py

```diff
@@ -595,7 +576,7 @@ def _try_recover(self):
                 ev = self.optimizer.remote_evaluators[i]
                 w = ev
             try:
-                ray_get_and_free(obj_id)
+                ray.get(obj_id)
```

Comment:
This is an error I think. If you check our branch releases/0.6.6 I cherry picked the PR that introduced ray_get_and_free.

Reply:
@Edilmo I originally made this change particular against ray 0.6.6, but I double confirmed this should be the only place that may use your cherry-picked PR. You're right, I changed to accommodate your cherry-picked PR. Thanks for checking!